requirements.txt

```
@@ -0,0 +1,26 @@
+# Dev packages
+--editable git+https://github.com/inducer/pymbolic.git#egg=pymbolic
+#--editable git+https://github.com/inducer/islpy.git#egg=islpy
+#--editable git+https://github.com/inducer/pyopencl.git#egg=pyopencl
+--editable git+https://github.com/inducer/loopy.git#egg=loopy
```

```suggestion
--editable git+https://github.com/inducer/loopy.git#egg=loo.py
```
(This is necessary for now at least.)

---

Fixed in d26e39e.